source/plugin/scheduler.rst

```diff
@@ -83,7 +83,8 @@ Using the ``Task.Builder``, you can specify other, optional properties, as descr
 | synchronization | async()                 | A synchronous task is run in the game's main loop in series with the     |
 |                 |                         | tick cycle. If ``Task.Builder#async`` is used, the task will be run      |
 |                 |                         | asynchronously. Therefore, it will run in its own thread, independently  |
-|                 |                         | of the tick cycle, and will not be thread safe with game data.           |
+|                 |                         | of the tick cycle, and may not safely use game state.                    |
+|                 |                         | (See :ref:`asynchronous-tasks`.)                                         |
```

Comment:
Any reason for using this and not just \`Asynchronous Tasks\`_ ?
With this, you unnecessarily create the need for a reference to `_asynchronous-tasks` below.